### PR TITLE
Feat/45 offices card

### PR DESCRIPTION
**Pull request by arabellamejorada**
read me pls:

- finished the structure of the office cards, but I need suggestions from u bossings on how to polish the bottom part of the component (sa cutout part) cause it looks like this and idk what to do na huhu:
image

- also placed the View Office button below the name and position cause if I place it at the bottom right, it will ruin the structure of the image cutout. open for suggestions for this too!

(ps. a bit challenging ticket frfr. would love to hear from ur pr reviews po)

### CODE REVIEW

**Comment by shunindesuuu** [comment]
hello ara! here are my suggestions lang

for the image:
- i think this is an issue sa cutout? so i guess pwede siguro edit ang image na naga overlap sa bottom left

for the button:
- try changing the z-index para di sila in the same layer with the image

File: src/components/office-cards/BaseCard.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import Button from '@/components/ui/Button';
+
+const BaseCard: React.FC<{
+  imageSrc: string;
+  name: string;
+  title: string;
+}> = ({ name, title, imageSrc }) => {
+  return (
+    <div className="relative flex items-center bg-blue text-white w-full max-w-[900px] rounded-[30px] md:max-w-[900px] transition-all duration-300 overflow-visible" style={{ height: '400px' }}>
```
Comment (shunindesuuu) [inline src/components/office-cards/BaseCard.tsx:11]: change this to tailwind

File: src/components/office-cards/BaseCard.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      {/* Officer Image */}
+      <div className="absolute left-0 bottom-0 w-[50%] h-auto" style={{ transform: 'translateY(-.1%)' }}>
```
Comment (shunindesuuu) [inline src/components/office-cards/BaseCard.tsx:13]: this one as well

**Review by miggyalino — CHANGES_REQUESTED** [review]
Please delete OSG, OSP, OST, OVP Card component. No need na since ang difference lang kay ang props being passed in. Ipa pass in lang ni sa mag gamit sa component.

**Review by shunindesuuu — APPROVED** [review]

**Review by miggyalino — APPROVED** [review]

**Pull request merged**